ui/server/src/main/scala/pl/touk/nussknacker/ui/component/ComponentService.scala

```diff
@@ -199,17 +202,24 @@ class DefaultComponentService private(config: Config,
           processingType
         )
 
-        def createLinks(componentId: ComponentId, componentName: String, componentType: ComponentType) =
-          componentLinks
-            .filter(_.isAvailable(componentType))
-            .map(_.toComponentLink(componentId, componentName))
+        def createLinks(componentId: ComponentId, component: ComponentTemplate): List[ComponentLink] = {
+          val componentLinks = componentLinksConfig
+            .filter(_.isAvailable(component.`type`))
+            .map(_.toComponentLink(componentId, component.label))
+
+          //If component configuration contains documentation link then we add base link
+          getComponentDoc(uiProcessObjects.componentsConfig, component)
+            .map(ComponentLink.createDocumentationLink)
+            .map(doc => componentLinks ++ List(doc))
```

Comment:
should it be last or first?

Reply:
Hmm, you're right - first will be better.